st2actions/tests/unit/policies/test_concurrency.py

```diff
@@ -88,12 +88,18 @@ def setUpClass(cls):
         loader.save_fixtures_to_db(fixtures_pack=PACK,
                                    fixtures_dict=TEST_FIXTURES)
 
+    # NOTE: This monkey patch needs to happen again here because during tests for some reason this
+    # method gets unpatcked (test doing reload() or similar)
```

Comment:
s/unpatcked/unpatched/

Reply:
Thanks, will fix those typos 

As far as the actual change goes, it's quite nasty. We've had those tests intermediy failing for a long time without being able to track down the root cause.

The problem is the asynchronous nature of the tests and that they only fail every now and then (usually on master where coverage which changes test timing a bit is enabled).

I'm still not 100% sure this will fix all the issues, but will see.